manifests/client.pp

```diff
@@ -152,18 +136,14 @@
     $_file_ensure = absent
     $_cron_ensure = absent
   }
-  if is_string($cron_hour) {
+  if $cron_hour.is_a(String) {
```

Comment:
That’s a strange syntax to use. Normally types are compared using the =~ operator. Here I would probably just use `$_cron_hour = Array($cron_hour, true)`, this will convert to a single element array if string or return umodified if array. You can save the if/else. https://help.puppet.com/core//current/Content/PuppetCore/typecasting.htm#Convertingdatastructures